3567/0/triangle.rb

```diff
@@ -0,0 +1,39 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+#:reek:ControlParameter:reek:UtilityFunction and :reek:FeatureEnvy and :reek:ControlParameter
```

Comment:
Here we have only your code, so fix these offenses with styling your code not with disables